entities/classcache/context.go

```diff
@@ -39,26 +39,42 @@ func RemoveClassFromContext(ctxWithClassCache context.Context, name string) erro
 	return nil
 }
 
-func ClassFromContext(ctxWithClassCache context.Context, name string, getter func(name string) (*models.Class, uint64, error)) (*models.Class, uint64, error) {
+type VersionedClass struct {
+	*models.Class
+	Version uint64
+	// TODO: we can pass error to check against
```

Comment:
question(non-blocking): change to `TODO-RAFT` ?

Reply:
i am actually planning to do all those todo, i've created, it was just reminder for me to not forget